src/administration/integrations/webhooks.md

```diff
@@ -11,7 +9,71 @@ launch automated user testing.
 $ platform integration:add --type=webhook --url=A-URL-THAT-CAN-RECEIVE-THE-POSTED-JSON
 ```
 
-Here you can see an example of the response:
+The webhook URL will receive a POST message for every "Activity" that is triggered, and the message will contain complete information about the entire state of the project at that time.  In practice most of the message can be ignored but is available if needed.  The most commonly used values are documented below.
+
+It's also possible to set the integration to only send certain activity types, or only activities on certain branches.  The CLI will prompt you to specify which to include or exclude.  Leave at the default values to get all events on all environments in a project.
+
+## Webhook schema
+
+### `id`
+
+A unique opaque value to identify the activity.
+
+### `project`
+
+The Project ID for which the activity was triggered.  Use this value if you want to have multiple projects POST to the same URL.
+
+### `type`
```

Comment:
I couldn't find a canonical list of all possible activities anywhere.  The ones below I found through either trial-and-error or grepping the Git source code.  @damz @ralt Please advise if I missed any.

Reply:
FYI: there's also [environment.merge-pr](https://lab.plat.farm/sdk/git/blob/master/platform_git/models/activities/code.py#L313) which is emitted only by the bitbucket integration when a pull request is merged in bitbucket.

Reply:
I confirmed that you've listed all other activity types.

Reply:
I believe that one I removed at Damien's recommendation.